lib/workers/repository/index.js

```diff
@@ -31,6 +49,7 @@ async function renovateRepository(repoConfig) {
       config.localDir || path.join(tmpDir, config.platform, config.repository);
     await fs.ensureDir(config.localDir);
     logger.debug('Using localDir: ' + config.localDir);
+    await setGitPrivateKey(config);
```

Comment:
Do you think we could do this as part of platform.initRepo instead? That way we do it immediately after we clone or fetch and be sure it's done before any commits.

Reply:
I found that `localDir` is ensured here, so I made the function call over here to make sure localDir is present

Reply:
But don't we call platform.initRepo after this too? i.e. we can be assured its done after the localDir is set

Reply:
yeah, you're right, makes sense

Reply:
changed the location